cinema/tests/test_cinema_hall_api.py

```diff
@@ -43,9 +41,7 @@ def test_get_cinema_halls(self):
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(response.data[1]["name"], vip_hall["name"])
```

Comment:
use get() method instead of parenthesis everywhere to avoid KeyError like
`        self.assertEqual(response.data[1].get("name"), vip_hall.get("name"))
`